### PR TITLE
Docs: changelog review.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: changelog review.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,37 +9,37 @@ We are happy to announce the release of CKEditor 5 v27.0.0.
...
-* **[engine](https://www.npmjs.com/package/@ckeditor/ckeditor5-engine)**: Introduced bubbling of `view.Document` events, similar to how bubbling works in the DOM. Bubbling allows listening on a view event on a specific kind of an element, hence simplifying code that needs to handle a specific event for only that element (e.g. `enter` in `blockquote`s only). Read more in the documentation: **\[TODO\]**. Closes [#8640](https://github.com/ckeditor/ckeditor5/issues/8640). ([commit](https://github.com/ckeditor/ckeditor5/commit/5527283324ad8bef5231acde0e49f9fc78df9c90))
+* **[engine](https://www.npmjs.com/package/@ckeditor/ckeditor5-engine)**: Introduced bubbling of the `view.Document` events, similar to how bubbling works in the DOM. Bubbling allows listening on a view event on a specific kind of element, hence simplifying code that needs to handle a specific event for only that element (e.g. `enter` in `blockquote`s only). Read more in the documentation: **\[TODO\]**. Closes [#8640](https://github.com/ckeditor/ckeditor5/issues/8640). ([commit](https://github.com/ckeditor/ckeditor5/commit/5527283324ad8bef5231acde0e49f9fc78df9c90))
```
Comment (pomek) [inline CHANGELOG.md:40]:
```suggestion
* **[engine](https://www.npmjs.com/package/@ckeditor/ckeditor5-engine)**: Introduced bubbling of the `view.Document` events, similar to how bubbling works in the DOM. Bubbling allows listening on a view event on a specific kind of element, hence simplifying code that needs to handle a specific event for only that element (e.g. `enter` in `blockquote`s only). Read more in the documentation: [The event system deep-dive guide](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/event-system.html). Closes [#8640](https://github.com/ckeditor/ckeditor5/issues/8640). ([commit](https://github.com/ckeditor/ckeditor5/commit/5527283324ad8bef5231acde0e49f9fc78df9c90))
```

Comment (godai78) [reply CHANGELOG.md]: We added that simultaneously :)

**Comment by AnnaTomanek** [comment]
Added some corrections. 3 things still to do (potentially later):

*   In the new features part, the links should better go to the docs and not to the issues.
*   I am wondering why we list one bug fix in the highlights (honestly, we can remove it from there, it's minor) but it's not then listed in the "Bug fixes" part below. Instead, there are 2 other bug fixes there :D So did we really fix one bug or not really?
*   Also I'm not sure about adding this:

> \* **\*\*\[language\](**https://www.npmjs.com/package/@ckeditor/ckeditor5-language**)\*\***: Added the \`'textPartLanguage'\` toolbar dropdown to support setting a text part language. Closes \[#8989\](#8989).

I mean, this release has just introduced this feature. What's so special about adding a dropdown for it? Shouldn't we just focus on the fact that we introduced the new feature (with all its parts)?

**Comment by pomek** [comment]
> (honestly, we can remove it from there, it's minor)

👍 

> Also I'm not sure about adding this:

👍

**Comment by godai78** [comment]
> In the new features part, the links should better go to the docs and not to the issues.

Not sure about it, we would have some links to docs and some links to GH (not everything is documented). But we can try introducing this.

**Comment by AnnaTomanek** [comment]
Docs links will need to be replaced once the docs are published. For now, I rewrote the text part language entry. The last thing missing is Elsevier's bug.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,43 +7,42 @@ Changelog
...
-* **[language](https://www.npmjs.com/package/@ckeditor/ckeditor5-language)**: Added the `'textPartLanguage'` dropdown button to support the text part language. Closes [#8989](https://github.com/ckeditor/ckeditor5/issues/8989).
+* **[language](https://www.npmjs.com/package/@ckeditor/ckeditor5-language)**: Added support for setting the text part language. Closes [#8989](https://github.com/ckeditor/ckeditor5/issues/8989).
 
 ### Bug fixes
 
 * **[engine](https://www.npmjs.com/package/@ckeditor/ckeditor5-engine)**: `DataController#toView()` should have a default value for the `options` parameter. Closes [#9293](https://github.com/ckeditor/ckeditor5/issues/9293). ([commit](https://github.com/ckeditor/ckeditor5/commit/f77a3d57bddb96ae3280736f974cfd0b148611cb))
+* **[highlight](https://www.npmjs.com/package/@ckeditor/ckeditor5-highlight)** The remove highlight button now also gets disabled along with the main highlight command. Closes [#9174](https://github.com/ckeditor/ckeditor5/issues/9174).
```
Comment (pomek) [inline CHANGELOG.md:45]:
```suggestion
* **[highlight](https://www.npmjs.com/package/@ckeditor/ckeditor5-highlight)** The remove highlight button now also gets disabled along with the main highlight command. Closes [#9174](https://github.com/ckeditor/ckeditor5/issues/9174). ([commit](https://github.com/ckeditor/ckeditor5/commit/04acdfec9e7ee4b38daa1ef372e201f535d960fc))
```

**Pull request merged**